repo/packages/C/cassandra/0/config.json

```diff
@@ -91,6 +97,11 @@
           "type": "string",
           "default": "dcos"
         },
+        "binary": {
+          "description":"Cassandra binary",
+          "type":"string",
+          "default":"http://downloads.mesosphere.com/cassandra-mesos/artifacts/0.2.0-1/cassandra-mesos-0.2.0-1.tar.gz"
+        },
```

Comment:
Thanks for the feedback @jsancio. The intention behind doing this is basically allow integration tests over-ride the binary location. Doing so sets us free from maintaining a separate testing branch.

Is there any way I can override resource.json properties via options.json or some other mechanism ?